website/src/index-page.tsx

```diff
@@ -498,10 +502,31 @@ function ButtonLink({
 function TextLink({
   children,
   className,
+  noAnchor,
+  href,
   ...props
-}: ComponentProps<typeof Anchor>) {
+}: ComponentProps<typeof Anchor> & { noAnchor?: boolean }) {
+  const router = useRouter()
+  if (noAnchor) {
+    return (
+      <span
+        onClick={(e) => {
+          e.preventDefault()
+          router.push(href)
```

Comment:
also `href` should have `basePath` suffix

Reply:
What do you mean by that?

Reply:
I was wrong, `basePath` from `next.config` is appended automatically